contracts/Staking.sol

```diff
@@ -35,7 +36,10 @@ contract Staking {
         _;
     }
 
-    constructor() {}
+    constructor(uint32 minNumValidators, uint32 maxNumValidators) {
```

Comment:
By the way, add checking not if _minimumNumValidators exceeds _maximumNumValidators in constructor is user-friendly